lib/vagrant-libvirt/templates/private_network.xml.erb

```diff
@@ -33,6 +33,9 @@
             <bootp file="<%= @network_dhcp_bootp_file %>" />
           <% end %>
         <% end %>
+        <% if @network_dhcp_default_lease_time %>
+            <lease expiry='@network_dhcp_default_lease_time' unit='hours'/>
```

Comment:
From the issue discussion, this looked like it should be inside the range tag, while here it is only inside the dhcp tag. Appears would need to change how the range tag is closed to add this.

Reply:
I think it'll require changing the range tag as well to remove the trailing `/>` to prevent it being closed before this as well. Presumably this means the current bootp XML is ignored as well

Reply:
True. If my XML is not off, commit 4c25b52 should fix this...